### PR TITLE
Removes the Gatfruit from Botany

### PR DESCRIPTION
**Pull request by SamHPurp**
Removes the Gatfruit tree from Botany, removing their ability to grow .357 Revolvers....

All reactions to this when seen/grown in game has been one of complete and utter disbelief in IC and OOC overwhelmingly.

:cl: Purpose2
tweak: Removes the Gatfruit tree from Botany.
/ :cl:

### CODE REVIEW

**Comment by LightFire53** [comment]
Please. How do you realistically grow firearms anyway.

**Comment by scrubmcnoob** [comment]
Now this is just anti-american
smh all of you are uncultured swine
I got ten gatfruit in my basement growing all varieties of different gats.

**Comment by MrFroztee** [comment]
> How do you realistically grow firearms anyway.

Are you trying to apply logic to this fictional silly spaceman game

**Comment by MrFroztee** [comment]
`rarity = 60 // Obtainable only with xenobio+superluck.`

Sounds like a pretty low chance for a high reward. If Gatfruit, name being obvious in what they supply, provided some other means of produce, with a .357 being about as rare as finding a storage implant in maintenance, I wouldn't really see a problem with it.

Just modify the typical produce to be snap pops or an actual Gatfruit.

**Comment by Crazylemon64** [comment]
@SamHPurp Could you provide reasoning, please?

**Comment by Crazylemon64** [comment]
@MrFroztee `rarity` doesn't control how seldom the mutation occurs, it controls how much the seed is worth when shipped to centcomm.

**Comment by MrFroztee** [comment]
> it controls how much the seed is worth when shipped to centcomm

Well that's just silly, now, isn't it? I wish there was more fun stuff like this in botany, as overpowered as a revolver grown on a vine might be considered. I'd code in less extreme stuff like that myself, if I comprehended it.

**Comment by Crazylemon64** [comment]
It doesn't actually appear possible to acquire gatfruit outside of adminbus. Why are you doing this?

**Comment by SamHPurp** [comment]
@Crazylemon64 I've seen them in several rounds, so it can't be just adminbus?

Botany growing revolvers is just the epitome of silly. 

Everyones reactions ingame IC and OOCly have been one of complete and utter disbelief.

**Comment by Anoonki** [comment]
@Crazylemon64 
IIRC, xenobio can give botany an item that they can *somehow* make into gatfruit, or grow gatfruit as a result of getting.

**Comment by Crazylemon64** [comment]
Oh! Silver core to get a gatfruit's "Fruit", which can then be given to botany. I see.

**Comment by MrFroztee** [comment]
@Crazylemon64 I believe that's why it says it needs Xenobio's assistance.

**Comment by uraniummeltdown** [comment]
Lame, if Science are working with Botany to grow Gatfruit then perhaps Sec should raid them? Either way it's a pretty rare drop from a silver slime core, given that there is tons of other food it can drop.

**Comment by Necaladun** [comment]
Haven't seen it in game yet, but the idea of botany making 357s is honestly just noooooope

**Comment by Crazylemon64** [comment]
If you're concerned about this being out of place, how about renaming these to "Pea Shooters"?

**Comment by Danielhuren** [comment]
i feel i should point this out since no ones figured it out yet its not gonna be botany that grows gatfruit its gonna be sci

**Comment by Crazylemon64** [comment]
yeah sharing is not done enough and science can build literally anything

**Comment by SamHPurp** [comment]
A fair point. 

Everytime I've seen this done, it was done by the RD in seclusion. With no sharing going on whatsoever.

So unfortunately I still stand by the decision to drop this from the code base. It adds nothing, and is only utilised by an already overstuffed Science team.

**Comment by KingPhilipIII** [comment]
>Fun police

**Comment by HereticChurch** [comment]
I'd like to see the weapons produced replaced with plant guns called pea shooters and weaken them by a large margin, at the very least. That at least has them make some sense. I'd prefer them changed over removed, personally.

**Comment by Necaladun** [comment]
In general I'd prefer them changed over removed, but botany particularly has so much bloat atm I don't see it as a huge loss to  just remove.

Otoh, I haven't seen it as a problem yet, so no rush.

**Comment by Danielhuren** [comment]
again its literally only sci that can get them and botany might as well be screwed its pointless unless you make a way to get them that doesn't use xenobio so they might aswell be removed cause if you add it to botany people will literally abuse the fuck out of them and as a scientist out of all the things i wouldent give them 357s is it

**Comment by Danielhuren** [comment]
and if you do give it to botany replace it with 38s if you must give it to them tree the grows guns

**Comment by Ionward** [comment]
Instead of just outright removing this, change how its obtained if its such a problem. The excuse of "it's surprising" or "it doesn't make sense" is bullshit in SS13, and in botany especially. There are flowers that light you on fire if you get hit with them. You can grow meat and eggs, combustible lemons, golden apples, and don't even get me started on the contents of strange seeds. If the RD hoarding .357s is such an issue (he can already hoard other guns), change where they come from, make their seeds a thing mining finds, or have some gateway missions contain them. Outright removing things is never fun.

**Comment by SamHPurp** [comment]
So just to prove a point, I just tried this on Paradise on the live server.

They cost 13 TC for a Traitor to buy them... and with fairly half-assed work as a Scientist, we were able to make upwards of 50 of these guns in 90 minutes... and I'm a very very bad botanist / xenobiologist.

A competent one will do this much much better.

They are fully loaded, and as long as you don't break them out of the pod... you can fit them into small boxes.

Basically I walked around with about 50 odd of them, and handed them out on the shuttle.

Balance wise this is a nightmare.

~~Muh immurshuns wise this is a nightmare.~~

**Comment by Jovaniph** [comment]
Way to prove a point... You have my vote.

**Comment by AndriiYukhymchak** [comment]
this is breaking the low rp we have

**Comment by uraniummeltdown** [comment]
Don't remove it that's boring, if it's really such a problem just blacklist it from xenobio and make the seed have a low chance of spawning in one of the rooms on the mining asteroid

**Comment by Dinarzad** [comment]
It doesn't need to be ON the mining asteroid, there's no non-antag reason to be able to grow fucking guns.
Botany has plenty of ways to fuck around and/or murder people if that's their desire, They can stand to be slightly more "Boring" by losing a single plant.

**Comment by jughu** [comment]
Basically its easy anyways let me roll trough
As a fact you cant chamge the maturation speed well thats a -
-But infact you can add multiple harvests to it from one of the multiple harvest trait plants this will remove the replant seed factor
You can get yield 10 from coffee seeds
You get production 1 from wheat(imporant for multiple harvest!! Since this highly increases production)
Have the weed eating trait so you dont need to care about the wheat
Use bees(maybe for  secret yield/potency increase yes infact it does increase potency over a 100 plus it can make plants hold 100 fruits)
Gatfruit big harvests are extremly easy with the plant manipulator i suggest taking making gatfruit damage based of potency and also one use gun so potency 50 is example 30 brute since orginally its 60

**Comment by Danielhuren** [comment]
so heres my idea after trying it multiple times ona private server from round start as a solo person turns out  the chances of getting gat fruit from senobio early enough to make a difference is some were around  1 in 5 shifts if your alone with help im betting its probly 50 50 so i have less of a problem now that i spent 7 or 8 hours testing it on a private server and yes i have no life but my issue with gatfruit has shifted from one of ballance to one of why can sci grow this but botany cant and i would have no problem with possibly introducing a traitor seed crate for botany for like 15 tc that has this crap in it cause agian this is a botony thing yet only sci can reliably get it 50 percent of the time WITH help what happened when me and sam did it on server was a lucky fluke i think but i wil maintain i no longer think its a balance issue as to the fruit itself that requires it to be removed although i would remove it if no one wants to dedicate the time to make a botinist traitor item

**Comment by SamHPurp** [comment]
@Danielhuren - I agree with the idea that legitimately non-antags shouldn't be growing guns, so moving it towards a traitor item isn't an entirely unappealing idea.

@Fox-McCloud  - You like to silently thumbs down my PRs, so as a maintainer, I'd appreciate your input into why you have silently thumbs'd this down. Disagree with the any changes altogether? Or just the removal?

@uraniummeltdown - Most people didn't even know this existed, so we're certainly not making anything 'more boring'.

@Dinarzad - I agree that placing this on the mining asteroid instead is similarly dull, dull dull.

@jughu - Where were you when I was testing this? If I had a competent botanist at my side we'd have really blown some heads. As for altering the damage output, I'd say at this point if we were to do this, it'd be better to **massively** reduce the damage, get a new sprite, separate them out from the 13TC Traitor Item and call them something else, "Peashooter" mentioned above is legitimate.

Ultimately however, I don't see the sense in growing revolvers, especially as a non-antagonist, so I still stand by my straight removal of this. I do concede that the 'conversion to peashooters' idea is an interesting one, the fact that nobody really knew about this before now means nobody is going to miss this, and ultimately does this add anything? I don't think so.

Welcome additional feedback.

**Comment by KingPhilipIII** [comment]
Keep in mind gatfruit are incredibly hard to care for so if the botanist fucks up just a little poof they're gone. Making them a botanist tator item and illegal to grow isn't a bad idea in my opinion

**Comment by SamHPurp** [comment]
@KingPhilipIII one gaia branch and you can ignore it though.

**Comment by KingPhilipIII** [comment]
I know, but that would require the cooperation of chemistry, which doesn't always happen, and for you to hide your growing death plants from Sec, because they take a while to grow. Not including major gene modification. If it were a traitor item make it impossible to mod or summin.

**Comment by Draconic-saint** [comment]
While the idea of growing guns is hilarious to me, I can see the issue.
I'd propose a split between a botany/xeniobio/"All organic no GMO" version, that grows nerf guns, and a "SUPER GENEMODDING 8000" version that produces actual live guns, for the evil evil syndicate.

Or the otherway around if you're into that whole "NT are the baddies" type deal. _goddamn traitors..._

**Comment by jughu** [comment]
the gun damage should be poteeeency based

**Comment by ExitGame** [comment]
Fun police much?
The gatfruit is already extremely hard to get via silver slime core, I think it's in the 1:1000 levels, we don't have to remove everything interesting. Not many even know that you can get this fruit via a silver slime core.

**Comment by Vivalas** [comment]
I really don't like this, if someone goes through the trouble then let them have their fun, crazy and wacky things make SS13 fun.

**Comment by mrturkeytoe** [comment]
I'm personally against this, I like wacky random fun shit like this.

**Comment by jughu** [comment]
Dont remove the gatfruit just refactor it like
Yield how much ammo it holds
Potency how strong the bullet it
Endurance chance that the gatfruit auctally include a gun

Always harvests 1
Seeds can be multiplied by the extractor of course

And heres yer balance

**Comment by SamHPurp** [comment]
Thanks for all your feedback, you may not feel necessarily it is the case but I have thought long and hard about making the sort of tweaks discussed above.

With all the proposed tweaks etc, this then becomes entirely an entirely different thing. Enough so that a PR to remove this, and a separate PR to 'add' this conceptual item down the line for whomever desires to do so, seems the logical path.

With one Head of Staffs agreement thus far, and my strong feeling that it should continue with being removed, I'll remain on this PR in its current state being the one considered by the Heads/Maintainers.
